<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/paypal.svg</normaloff>Images/paypal.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(14, 102, 22);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Header">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 61, 61);
border-radius: 8px;
position: absolute;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="font">
             <font>
              <family>Inter</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Senha incorreta</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_closeError">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	border-radius: 5px;
	background-image: url(:/Close/Images/X.png);
	background-position:center;
	background-color: rgb(255, 26, 26);

}

QPushButton:hover{
	background-color: rgb(50, 50, 50);
	color: rgb(200, 200, 200);
}
QPushButton:pressed{
	background-color: rgb(35, 35, 35);
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_logo">
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QLabel" name="logo">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>0</y>
            <width>50</width>
            <height>50</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Logo/Images/paypal.svg&quot; height=&quot;50&quot; width=&quot;50&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="logoTitle">
          <property name="geometry">
           <rect>
            <x>55</x>
            <y>10</y>
            <width>101</width>
            <height>38</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PSWRD
MANAGER</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <zorder>logoTitle</zorder>
         <zorder>logo</zorder>
        </widget>
       </item>
      </layout>
      <zorder>frame_logo</zorder>
      <zorder>frame_error</zorder>
      <zorder>frame</zorder>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="loginArea">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <widget class="QLineEdit" name="inputText_Email">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>220</y>
            <width>220</width>
            <height>50</height>
           </rect>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(14, 78, 20);
	border-radius: 5px ; 
	padding: 15px;
	background-color: rgb(254, 254, 254);
}

QLineEdit:hover{
	border:2px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:2px solid  rgb(14, 78, 20); 
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>50</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Informe seu e-mail</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_slogan">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>301</width>
            <height>91</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="slogan">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>221</width>
             <height>81</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;color:#FFFFFF&quot;&gt;Manage Your &lt;br/&gt;Passwords&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="slogan_2">
           <property name="geometry">
            <rect>
             <x>185</x>
             <y>35</y>
             <width>101</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;color:#FFFFFF&quot;&gt;Faster&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="avatar">
          <property name="geometry">
           <rect>
            <x>155</x>
            <y>110</y>
            <width>90</width>
            <height>90</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	
	background-image: url(:/Avatar/Images/profile circled.svg);
	background-repeat: no-repeat;
	background-position: center;
	border-radius: 30px;
}
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="inputText_Password">
          <property name="geometry">
           <rect>
            <x>95</x>
            <y>280</y>
            <width>220</width>
            <height>50</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit{
	border: 2px solid rgb(14, 78, 20);
	border-radius: 5px ; 
	padding: 15px;
	background-color: rgb(254, 254, 254);
}

QLineEdit:hover{
	border:2px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:2px solid  rgb(14, 78, 20); 
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Informe sua chave de criptografia</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_newDf">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>380</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(254, 254, 254);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(11, 82, 18);
	border: 2px solid rgb(8, 50, 12);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgba(64, 155, 71, 250);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(10, 65, 15);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(24, 152, 35);
	border-radius: 5px;
}

</string>
          </property>
          <property name="text">
           <string>ADICIONAR NOVO 
BANCO</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="viewDFButton">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>385</y>
            <width>161</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color: rgb(14, 78, 20);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border: 2px solid rgb(8, 50, 12);
}

QPushButton:hover{
	background-color: rgb(17, 99, 24);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
	border: 2px solid rgb(8, 50, 12);
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 75 10pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border: 2px solid rgb(8, 50, 12);
	border-radius: 5px;
}

</string>
          </property>
          <property name="text">
           <string>VISUALIZAR BANCO</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="QTImages.qrc"/>
 </resources>
 <connections/>
</ui>
